tools/base/requirements.in

```diff
@@ -1,16 +1,17 @@
-aio.functional
-aio.subprocess
-aio.tasks
+abstracts>=0.0.12
+aio.functional>=0.0.9
+aio.subprocess>=0.0.4
+aio.tasks>=0.0.4
```

Comment:
@phlax Given the recent `npm` fiasco where a package owner screwed up their code intentionally causing those using `>=` for the deps, should we be using `>=`? See https://research.swtch.com/npm-colors for more detail. Or are we covered by the specific pinning in the `requirements.txt` below?

Reply:
we are kinda covered - the requirements.txt file is compiled from this file, so the actual requirements only change when recompiled etc

i had some conversation with @kfaseela about this, and what the best strategy for pinning is - im not totally sure of the best approach, but my general feeling is to pin the internal deps `>=` at any given time as that is what is currently known to work